### PR TITLE
Separate HoS and Armory access

### PR DESCRIPTION
**Pull request by JORJ949**
## About the PR 
Removes access_armory from all IDs except HoS and NTSO
Changes all armory crates, devices and doors to use access_armory rather than access_maxsec
Adds an Armory Access spawner for mapping

## Why's this needed? 
Seperation of accesses, allows jobs to be made with armory but not HoS access (e.g. the NTSO, which got HoS access but only should really get armory)
In addition the access_armory was unused

## Changelog 

```changelog
(u)JORJ949
(+)HoS and Armory access have been separated, presently only affects the NTSO who no longer gets HoS access. Please report any other changes as a bug.
```

### CODE REVIEW

**Comment by JORJ949** [comment]
> Not a fan of putting armory access in the ID computers

I put it in the same way as maxsec is implemented, it shouldnt show because of the later line removing maxsec and armory access listings

File: code/obj/machinery/computer/card.dm [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	var/list/engineering_access_list = list(access_external_airlocks, access_construction, access_engineering, access_engineering_storage, access_engineering_power, access_engineering_engine, access_engineering_mechanic, access_engineering_atmos, access_engineering_control)
 	var/list/supply_access_list = list(access_hangar, access_cargo, access_supply_console, access_mining, access_mining_shuttle, access_mining_outpost)
 	var/list/research_access_list = list(access_medical, access_tox, access_tox_storage, access_medlab, access_medical_lockers, access_research, access_robotics, access_chemistry, access_pathology, access_researchfoyer, access_artlab, access_telesci, access_robotdepot)
-	var/list/security_access_list = list(access_security, access_brig, access_forensics_lockers, access_maxsec, access_securitylockers, access_carrypermit, access_contrabandpermit)
+	var/list/security_access_list = list(access_security, access_brig, access_forensics_lockers, access_maxsec, access_armory, access_securitylockers, access_carrypermit, access_contrabandpermit)
```
Comment (JORJ949) [inline code/obj/machinery/computer/card.dm:18]: Note that access_maxsec was ALREADY listed here.

File: code/obj/machinery/computer/card.dm [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 /obj/machinery/computer/card/New()
 	..()
-	src.allowed_access_list = civilian_access_list + engineering_access_list + supply_access_list + research_access_list + command_access_list + security_access_list - access_maxsec
+	src.allowed_access_list = civilian_access_list + engineering_access_list + supply_access_list + research_access_list + command_access_list + security_access_list - access_maxsec - access_armory
```
Comment (JORJ949) [inline code/obj/machinery/computer/card.dm:33]: access_maxsec and access_armory are removed from the ID computer HERE

**Comment by Tarmunora** [comment]
> > Not a fan of putting armory access in the ID computers
> 
> I put it in the same way as maxsec is implemented, it shouldnt show because of the later line removing maxsec and armory access listings. Added comments to the code to point this out.

ok idcomp is more cursed than I thought lmao

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**